### PR TITLE
Add a Changelog

### PR DESCRIPTION
**Pull request by johnhawkinson**
It's really hard to keep track of the versions, and we need a way. Here is one.

### CODE REVIEW

**Comment by mlissner** [comment]
I've been using the releases system for this, though I've been negligent of late. It has the nice advantage of providing zips of the code at that point in time along with the notes. https://github.com/freelawproject/recap-chrome/releases

**Comment by johnhawkinson** [comment]
I don't object to that, but I think you want a way to track changes as-you-go. Also,, clearly your way…hasn't been working.

Ideally every time you land a PR or a patch you update the changelog, and then when you cut the release there's no difficulty identifying what the changes were.

File: ChangeLog [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+Version 0.8.1-signed.1-signed
+* Released Sept. 28, 2010
+* Works with Firefox 3.0.9 - 4.0.*
+* This release of RECAP fixes an issue introduced by the newest
+  version of PACER.
+* In addition, RECAP will now capture documents from PACER pages when
+  users are logged in to both ECF and PACER.
+
+Version 0.7
+* Released Aug. 22, 201
```
Comment (audiodude) [inline ChangeLog]: nit: 201 => 2010?

**Comment by mlissner** [comment]
This one is now over a year old. I'm finding that using Github to do releases is working pretty well. I don't like throwing away work, but @johnhawkinson, how do you feel about closing this one without merging?

**Comment by mlissner** [comment]
Closing here, but happy to continue discussing if anybody wants to advocate for this.

**Pull request closed without merging**